### ISSUE TITLE
Assert and Expect

### ISSUE THREAD

**Issue by ahsanbabar147** [john-doherty/selenium-cucumber-js#53]
I want to assert a heading's text on a webpage. 
<h3 class='RTL'> Hi I am someone <h3>
I want to assert "Hi" from the heading.

I tried like this:
this.Then(/^I should see Hi$/m, function(){
	driver.wait(until.elementsLocated(by.css('#body-section > div > div.row > div')), 10000);
	return driver.findElement(by.css('h3.RTL')).then(function(e){
		expect(e.getText()).to.include('Hi');
	})
});

Please help...

You haven't provided helpful examples. I follow them but I get error.
1) Scenario: Login to your account - features\phptravels-login.feature:8
   Step: Then I should see Hi - features\phptravels-login.feature:12
   Step Definition: step-definitions\phptravels-login-steps.js:183
   Message:
     TypeError: obj.indexOf is not a function
         at Assertion.include (C:\Users\Cyborg\Documents\Nodejs\node_modules\selenium-cucumber-js\node_modules\chai\lib\chai\core\assertions.js:228:45)
         at Assertion.assert (C:\Users\Cyborg\Documents\Nodejs\node_modules\selenium-cucumber-js\node_modules\chai\lib\chai\utils\addChainableMethod.js:84:49)
         at C:\Users\Cyborg\Documents\Nodejs\node_modules\selenium-cucumber-js\step-definitions\phptravels-login-steps.js:192:16
         at ManagedPromise.invokeCallback_ (C:\Users\Cyborg\Documents\Nodejs\node_modules\selenium-cucumber-js\node_modules\selenium-webdriver\lib\promise.js:1341:14)
         at TaskQueue.execute_ (C:\Users\Cyborg\Documents\Nodejs\node_modules\selenium-cucumber-js\node_modules\selenium-webdriver\lib\promise.js:2950:14)
         at TaskQueue.executeNext_ (C:\Users\Cyborg\Documents\Nodejs\node_modules\selenium-cucumber-js\node_modules\selenium-webdriver\lib\promise.js:2933:27)
         at asyncRun (C:\Users\Cyborg\Documents\Nodejs\node_modules\selenium-cucumber-js\node_modules\selenium-webdriver\lib\promise.js:2793:27)
         at C:\Users\Cyborg\Documents\Nodejs\node_modules\selenium-cucumber-js\node_modules\selenium-webdriver\lib\promise.js:675:7
         at process._tickCallback (internal/process/next_tick.js:109:7)

**Comment by john-doherty** [comment]
Please clean up your post, it's hard to follow

**Comment by ahsanbabar147** [comment]
Could you please provide some examples of how to use assert in this?
Like we use assert.Equals(expected, actual) in JAVA.
This is what I'm doing at present:
I load a page-->I provide login credentials --> I click login button---> I am redirected to my account
----> I click My Profile button ---->I provide text in some fields in a form---->I submit the form.

I want to assert that there exists a text "Hi "in heading of that webpage.
I want to assert the successful response message that appears after submitting the form.
 
You can check yourself:
Website:  www.phptravels.net/login
username: user@phptravels.com
password: demomuser
